h2o-core/src/main/java/water/rapids/ast/prims/mungers/AstRelevelByFreq.java

```diff
@@ -87,6 +87,13 @@ static int[] getMapping(int[] domainOrder){
         return mapping;
     }
 
+    /**
+     * Take the top N domains and sort rest of the indexes ASC 
+     * @param domainOrder domain order 
+     * @param topN number of top N domains to keep unsorted
+     * @param domainSize if the domain size is higher than domainOrder, new indexes are created and sorted ASC
```

Comment:
What if the domainSize is lower or equal?

Reply:
I removed the `domainSize` parameter. It was no longer helpful.